k8s/configuration/secret.yaml

```diff
@@ -14,7 +14,8 @@ metadata:
   namespace: default
 type: Opaque
 data:
-  # Base64-encoded Agent API Key (obtained from MongoDB Cloud Manager)
+  # Base64-encoded Group ID and API Key (obtained from MongoDB Cloud Manager)
+  group-key: "<b64 encoded Group ID>"
```

Comment:
Will rename the Secret to `cloud-manager-credentials` just like `threescale-credentials`.
Will also rename `group-key` to `group-id` for clarity